### ISSUE TITLE
[Suggestion] Add babel-polyfill to the project

### ISSUE THREAD

**Issue by Enchufadoo** [egoist/vuepack#142]
Hi I was just testing this with JavaFx webview, I got an error when trying to run the site:

`TypeError: undefined is not a function (evaluating 'Object.assign')`

I solved it by installing babel-polyfill

I think I'm using a custom implementation of webkit that comes with the framework.
Thought you may wanted to know.

**Comment by egoist** [comment]
nah, I might only add `Object.assign` polyfill, you don't need all of babel-polyfill.
